### PR TITLE
ACQ-669 Code.org/about/leadership page updates

### PR DESCRIPTION
**Pull request by dmantonyuk**
## Links

https://codedotorg.atlassian.net/browse/ACQ-669
https://code.org/about/leadership
https://code.org/about/education-leadership-council

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
image

**Comment by dmantonyuk** [comment]
1. Pages merged
3. Text fixed
5. Guess https://code.org/about/education-leadership-council link in the left navigation is not necessary so also removed
7. Guess it would be good to make a redirect and made a redirect: 
   from: https://code.org/about/education-leadership-council   
   to:   https://code.org/about/leadership

File: pegasus/sites.v3/code.org/public/about/leadership/index.md.erb [hunk 1]
```diff
@@ -15,10 +15,13 @@ style_min: true
...
+## Education Leadership Council
 
-We also work with two leadership councils to inform our work. [Learn more here](/about/education-leadership-council).
+<%= view :about_headshots, people:DB[:cdo_leaders].where(kind_s:'council') %>
 
 ## We're grateful for the advice and help of so many leaders over the years:
 
 <%= view :about_people, people:DB[:cdo_leaders].where(kind_s:'grateful') %>
+
+## We're grateful for the advice and help of so many leaders over the years:
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/about/leadership/index.md.erb]: This heading can be removed, looks like a duplicate from the section above:
Screenshot 2023-07-24 at 2 09 20 PM

**Review by kelbyhawn — COMMENTED** [review]
One minor update needed, but otherwise looks great!

**Comment by dmantonyuk** [comment]
@kelbyhawn please check one more time

**Review by kelbyhawn — APPROVED** [review]

**Pull request merged**